guides/common/modules/proc_deploying-a-policy-on-a-host-using-ansible.adoc

```diff
@@ -15,7 +15,8 @@ ifndef::satellite[]
 For more information, see xref:Installing_the_OpenSCAP_plugin_{context}[].
 endif::[]
 include::snip_prerequisite-rhel-repositories.adoc[]
```

Comment:
This is really incorrect for upstream. In non-Katello you can't even sync content but in upstream you can't assume they run RHEL.

Reply:
I'm planning to work on this in another PR. Not the scope of this PR.

Reply:
-> https://github.com/theforeman/foreman-documentation/pull/3159